App crashed when i sorted by price lowest and clicked on the displayed image.

### Description
I was browsing through the info of different cities and clicked on Hyderabad. I clicked on the restaurant to check the available info, and different restaurants was displayed. I discovered a menu button and clicked to check what feature could be accessed. I discovered that I can sort for restaurants in different ways, I clicked on price lowest  and it worked. But when I clicked on the displayed restaurant to check the info attached to it, the app crashed.

### Expected behaviour

I expected to see the info about the restaurant sorted according to lowest price

### How to reproduce the bug

After signing up and logging in to the app 
- search for Hyderabad and click on it. ( A page will open displaying some features to access different Info's about the city.

- Click on restaurant below by the right side.(different restaurants will be displayed)

- Click on the three dot by the upper right and press the first feature "price lowest".

- Click on any of the sorted restaurant image and note the crash.

### Reproducing the crash
https://youtu.be/GGqfoMgvYXI

- Device used: Tecno spark K7
- Operating system: Android 7.0
- App version: 5.4.0
### Additional comment

```04-29 09:53:17.255 30142 30142 E AndroidRuntime: FATAL EXCEPTION: main
04-29 09:53:17.255 30142 30142 E AndroidRuntime: Process: io.github.project_travel_mate, PID: 30142
04-29 09:53:17.255 30142 30142 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.project_travel_mate/io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity}: java.lang.IllegalArgumentException: Path must not be empty.
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2724)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2789)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.-wrap12(ActivityThread.java)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1527)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:110)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:203)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6251)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1063)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:924)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Path must not be empty.
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.squareup.picasso.Picasso.load(Picasso.java:297)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity.populateViews(RestaurantDetailsActivity.kt:48)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity.onCreate(RestaurantDetailsActivity.kt:44)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6670)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2677)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	... 9 more```

<sub>04-29 09:53:17.255 30142 30142 E AndroidRuntime: FATAL EXCEPTION: main
04-29 09:53:17.255 30142 30142 E AndroidRuntime: Process: io.github.project_travel_mate, PID: 30142
04-29 09:53:17.255 30142 30142 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.project_travel_mate/io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity}: java.lang.IllegalArgumentException: Path must not be empty.
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2724)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2789)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.-wrap12(ActivityThread.java)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1527)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:110)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:203)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6251)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1063)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:924)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Path must not be empty.
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at com.squareup.picasso.Picasso.load(Picasso.java:297)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity.populateViews(RestaurantDetailsActivity.kt:48)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at io.github.project_travel_mate.destinations.description.RestaurantDetailsActivity.onCreate(RestaurantDetailsActivity.kt:44)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6670)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2677)
04-29 09:53:17.255 30142 30142 E AndroidRuntime: 	... 9 more</sub>

--- a/Android/app/src/main/java/io/github/project_travel_mate/destinations/description/RestaurantDetailsActivity.kt
+++ b/Android/app/src/main/java/io/github/project_travel_mate/destinations/description/RestaurantDetailsActivity.kt
@@ -45,7 +45,9 @@ class RestaurantDetailsActivity : AppCompatActivity() {
     }
 
     private fun populateViews (restaurant: RestaurantDetails) {
-        Picasso.with(this).load(restaurant.featuredImage).into(image_view)
+        if (restaurant.featuredImage.trim().isNotEmpty()) {
+            Picasso.with(this).load(restaurant.featuredImage).into(image_view)
+        }
 
         address.text = getString(R.string.rest_address, restaurant.address)
         avg_cost.text = getString(R.string.avg_cost, restaurant.avgCost.toString())

